tests/testthat/helper-preload.R

```diff
@@ -19,6 +19,22 @@ ap <- function(.name, .prefix) {
   paste0(.prefix, .name)
 }
 
+add_mock_sample <- function(.immdata, .sample_name, .meta = NA) {
```

Comment:
you can declare .meta without default value and later use if(!missing(.meta)){...} to use in condition, but your variant also works.